lib/pages/new_group/widget/expansion_participants_list.dart

```diff
@@ -23,7 +23,7 @@ class ExpansionParticipantsList extends StatefulWidget {
 }
 
 class _ExpansionParticipantsListState extends State<ExpansionParticipantsList> {
-  bool isExpanded = false;
+  bool isExpanded = true;
```

Comment:
Why do you change it, i set it default to be false because, when have a long list, it will auto scroll down a little bit, so user can't see group image, so i set default to false. Please check the case when have a lot of contacts also

Reply:
IMO we should found another way to solve that, we need to expand it by default

Reply:
I changed it because imo it was better UI now we removed settings tiles

Reply:
not working atm with many contacts

Reply:
wjhat is not working ?

Reply:
@sherlockvn it looks like it was a build error, it's working now

Reply:
![expandablelist](https://github.com/linagora/twake-on-matrix/assets/48354990/a6898733-e83b-4267-9b6a-c7ee3a8874cc)